app/main.py

```diff
@@ -2,5 +2,19 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    results_storage = {}
+
+    def inner_checker(*args) -> str:
+
+        key = args
```

Comment:
key is not a really descriptive name. You can leave `args` to show what exaclty you want a key in your dictionary to be